bramkouwen/Neatly-In-Motion
# Updating Products Page

## What does this PR do?
* It updates products.html 

## How do you check this PR?
* Pull down the branch
* Check the added content to products.html  

## Screenshots 
None.
